install.sh

```diff
@@ -0,0 +1,29 @@
+# Test for root
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root (sudo)"
+  exit
+fi
+
+# Copying the agent to its final destination
+cp -r jumper-logging-agent /opt
+cd /opt/jumper-logging-agent
```

Comment:
The script should exit at the same directory (a side effect of changing the current dir is not nice).

use `pushd` here and `popd` at the end, or simply add the appropriate arguments to the command to operate on a different directory (e.g, `virtualenv $DIR/venv`)